docs/editors-guide/mondo-website-editing.md

```diff
@@ -32,3 +32,17 @@ TODO: These instructions are out-of-date
 8. Control c to end session
 9. Commit changes to github:
 `git add -A;git commit -m "update webpage";git push`
+
+
+## Docker based instructions
+
+- Checkout the gh-pages branch and cd to it
+- Run `docker run --name mondo-docs -p "4000:4000" -p "35729:35729" --volume="$(pwd):/srv/jekyll:Z" -it jekyll/jekyll bash`
```

Comment:
I'm on a M2 Mac and get this warning message after running the above line
```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
bash-5.1# 
```

Although the container is running, to avoid confusion about the warning I'd suggest adding:
If using a Mac processor (M1, M2, etc.) run as: 
`docker run --platform linux/amd64 --name mondo-docs -p "4000:4000" -p "35729:35729" --volume="$(pwd):/srv/jekyll:Z" -it jekyll/jekyll bash`